operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py

```diff
@@ -0,0 +1,17 @@
+from unittest.mock import MagicMock, patch
+
+from operatorcert.entrypoints import ocp_version_info
+
+
+@patch("operatorcert.entrypoints.ocp_version_info.setup_argparser")
+@patch("operatorcert.entrypoints.ocp_version_info.pathlib.Path")
+@patch("operatorcert.entrypoints.ocp_version_info.ocp_version_info")
+@patch("operatorcert.entrypoints.ocp_version_info.json")
```

Comment:
Mocking json should not be needed.

Reply:
I tried it before, but it was throwing me TypeError.

Reply:
That's because you need to return valid json from `ocp_version_info` mock.

Reply:
@haripate Any update to this?

Reply:
I was on PTO. Just updated it.